testsuites/syncgateway/functional/tests/test_xattrs_grants.py

```diff
@@ -690,6 +691,56 @@ def test_missing_xattrs_key(params_from_base_test_setup, missing_type):
         assert raw_doc["_meta"]["xattrs"][user_custom_channel1] is None, "raw doc has _meta xattrs with user xattrs key missing on sgw config "
 
 
+@pytest.mark.channels
+@pytest.mark.syncgateway
+def test_xattrs_key_with_disabled_xattrs(params_from_base_test_setup):
+    """
+    @summary:
+    https://docs.google.com/spreadsheets/d/15zvscRgX2U2Q1xDpUwYU54G6tVmTv_lMDOSK8G-uzeo/edit#gid=793208277
+    covers #26
+    1. Disable xattrs and enable user xattrs key in sgw config
+    2. Verify SGW throws an error when starting the sgw
+    """
+
+    sg_url = params_from_base_test_setup["sg_url"]
+    cluster_config = params_from_base_test_setup["cluster_config"]
+    mode = params_from_base_test_setup["mode"]
+    sync_gateway_version = params_from_base_test_setup["sync_gateway_version"]
+    xattrs_enabled = params_from_base_test_setup["xattrs_enabled"]
+    flag = False
+
+    if sync_gateway_version < "3.0.0" or xattrs_enabled:
+        pytest.skip('SGW version is not 3.0 and above or xattrs has to be disabled')
+
+    user_custom_channel1 = "channel1"
+    sg_conf_name = "custom_sync/sync_gateway_custom_sync"
+    sg_config = sync_gateway_config_path_for_mode(sg_conf_name, mode)
+
+    temp_sg_config, _ = copy_sgconf_to_temp(sg_config, mode)
+    temp_sg_config = replace_string_on_sgw_config(temp_sg_config, "{{ user_xattrs_key }}", "")
```

Comment:
does this enabling user_xattrs_key? does it need a string or true value?

Reply:
string value